src/ui/components/models/todo.ts

```diff
@@ -0,0 +1,26 @@
+import { tracked } from "@glimmer/component";
```

Comment:
This would generally go in `src/data/models/todo.ts` or `src/ui/components/todo-item/-utils/todo.ts` (I personally prefer the former)...